zombienet_tests/functional/0004-slashing.toml

```
@@ -0,0 +1,46 @@
+[settings]
+timeout = 1600
+bootnode = true
+
+[relaychain.genesis.runtime.runtime_genesis_config.configuration.config]
```

```suggestion
[relaychain.genesis.runtime.configuration.config]
```

Kusama have a diff path to change the config.